CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileCopyrightText: 2019-2022 Second State INC
 
-cmake_minimum_required(VERSION 3.10.2)
+cmake_minimum_required(VERSION 3.15)
```

Comment:
Does this issue need to upgrade cmake version to fix?

Reply:
I think so. At least for release builders.
CMAKE_MSVC_RUNTIME_LIBRARY was added in 3.15 so on earlier versions of CMake this variable will not take effect